object-store/src/main/java/org/hypertrace/config/objectstore/DefaultObjectStore.java

```diff
@@ -120,7 +142,11 @@ protected DefaultObjectStore(
       configChangeEventGeneratorOptional.ifPresent(
           configChangeEventGenerator ->
               configChangeEventGenerator.sendDeleteNotification(
-                  context, object.getData().getClass().getName(), deletedConfig.getConfig()));
+                  context,
+                  this.buildClassNameForChangeEvent(object.getData()),
+                  this.buildDataFromValue(this.buildValueForChangeEvent(object.getData()))
```

Comment:
extra round of conversions here is a bug (the value for change event can't go into build data from value, and doesn't need to).

IdentifiedObjectStore has updated to the correct version of this on line 157

Reply:
Ah, mixed them up. 🤦‍♂️